test/common/helpers.test.js

```diff
@@ -0,0 +1,67 @@
+import {
+  refPush,
+} from '../../common/script/index';
+
+import {
+  $w,
+} from '../../common/script/helpers';
```

Comment:
With this no longer on the exported `api` object, I had to import it directly thus. Legit? If so, would it be preferable to import everything from `helpers.js`, or does it still make sense to import from `index` where we can?